CODEOWNERS

```diff
@@ -265,62 +276,70 @@ Kconfig*                                  @tejlmand
 /subsys/mpsl/                             @nrfconnect/ncs-dragoon
 /subsys/mpsl/cx/                          @ankuns @martintv
 /subsys/mpsl/fem/                         @ankuns @martintv
-/subsys/net/                              @rlubos
+/subsys/net/                              @nrfconnect/ncs-co-networking
 /subsys/net/lib/mqtt_helper/              @nrfconnect/ncs-cia
 /subsys/net/lib/azure_*                   @nrfconnect/ncs-cia @coderbyheart
 /subsys/net/lib/aws_*                     @nrfconnect/ncs-cia @coderbyheart
 /subsys/net/lib/ftp_client/               @MarkusLassila @tomi-font
 /subsys/net/lib/icalendar_parser/         @lats1980
-/subsys/net/lib/lwm2m_client_utils/       @rlubos @SeppoTakalo @juhaylinen
+/subsys/net/lib/lwm2m_client_utils/       @nrfconnect/ncs-co-networking @SeppoTakalo @juhaylinen
 /subsys/net/lib/nrf_cloud/                @plskeggs @jayteemo @glarsennordic
```

Comment:
```suggestion
/subsys/net/lib/nrf_cloud/                @nrfconnect/ncs-nrf-cloud
```

Reply:
Please also change:
/tests/subsys/net/lib/nrf_cloud/ @tony-le-24
to:
/tests/subsys/net/lib/nrf_cloud/ @nrfconnect/ncs-nrf-cloud